incubator/drone/README.md

```diff
@@ -0,0 +1,65 @@
+# Drone.io
+
+[Drone](http://readme.drone.io/) is a Continuous Integration platform built on container technology. Every build is executed inside an ephemeral Docker container, giving developers complete control over their build environment with guaranteed isolation.
+
+## Introduction
+
+This chart stands up a Drone. This includes:
+
+- A [Drone Server](http://readme.drone.io/admin/installation-guide/) Pod
+- A [Drone Agent](http://readme.drone.io/admin/installation-guide/) Pod
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
```

Comment:
The PV is a Prerequisite only if we use the local `sqllite` ,if not it isn't. Do we want to add that clause?